code/game/objects/structures/watercloset.dm

```diff
@@ -506,7 +506,7 @@
 /obj/item/taperoll/bog
 	name = "toilet paper roll"
 	icon = 'icons/obj/watercloset.dmi'
-	desc = "A unbranded roll of standard issue two ply toilet paper. Refined from carefully rendered down sea shells due to SolGov's 'Abuse Of The Trees Act'."
+	desc = "A unbranded roll of standard issue two-ply toilet paper. Refined from carefully rendered-down seashells due to the Terran Confederacy's 'Abuse Of The Trees Act'."
```

Comment:
This lore change doesn't make sense to me. Aren't the TC very much "fuck alieums, fuck the planet"? Can't see them passing a "Abuse of The Trees" act.
Glloyd is our resident lore master, his input into bog roll lore would be appreciated here

Reply:
he doesn't know what the three seashells are for

Reply:
Changing it to TC is fine by me, the less SolGov the better. It's a Demolition Man reference (i assume) anyways.